### PR TITLE
Implemented logic to handle the case when product SKU is not present on  the server (#245fb8k).

### PR DESCRIPTION
**Pull request by meet-aniket**

### CODE REVIEW

File: src/views/OrderDetail.vue [hunk 1]
```diff
@@ -203,6 +208,13 @@ export default defineComponent({
    this.fetchFacilities();
   },
   methods: {
+    async listMissingSkus() {
+      const missingSkuModal = await modalController.create({
+        component: MissingSkuModal,
+        componentProps: { 'missingSkus': this.ordersList.unidentifiedProducts }
```
Comment (adityasharma7) [inline src/views/OrderDetail.vue:214]:
```suggestion
        componentProps: { 'unidentifiedProductItems': this.ordersList.unidentifiedProductItems }
```

File: src/store/modules/order/OrderState.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 export default interface OrderState { 
     list: {
       items: any,
-      original: any
+      original: any,
+      unidentifiedProducts: any,
```
Comment (adityasharma7) [inline src/store/modules/order/OrderState.ts:5]:
```suggestion
      unidentifiedProductItems: any,
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/locales/en.json [hunk 3]
```diff
@@ -36,7 +36,7 @@
   "Purchase order": "Purchase order",
   "Purchase orders": "Purchase orders",
   "Ready to create an app?": "Ready to create an app?",
-  "REVIEW": "REVIEW",
+  "Review": "REVIEW",
```
[the remark this answers is not in the document]
Comment (meet-aniket) [reply src/locales/en.json:40]: Made changes as per your comments Sir.

File: src/store/modules/order/OrderState.ts (hunk 2 above)
Comment (meet-aniket) [reply src/store/modules/order/OrderState.ts]: Changed Sir.

File: src/views/OrderDetail.vue (hunk 1 above)
Comment (meet-aniket) [reply src/views/OrderDetail.vue]: Changed Sir.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**